src/api/handlers/guild.ts

```diff
@@ -769,13 +769,9 @@ export function getGuild(guildID: string, counts = true) {
 }
 
 /** Returns the guild template if it exists */
-export function getGuildTemplate(
-  guildID: string,
-  templateCode: string,
-) {
-  return RequestManager.get(
-    `${endpoints.GUILD_TEMPLATES(guildID)}/${templateCode}`,
-  ) as Promise<Template>;
+export async function getTemplate(templateCode: string) {
+  const template = await RequestManager.get(endpoints.GUILD_TEMPLATE(templateCode)) as GuildTemplate;
+  return structures.createTemplate(template);
```

Comment:
I would like to await this into a const and return the const or atleast return await but i personally hate return await.

The await is necessary for stack tracing otherwise it gets realllllllllllly annoying to figure out what line caused the error.